src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -270,4 +270,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.IdentityModel.Abstractions" Version="6.22.0" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == '$(TargetFrameworkNet6)' Or '$(TargetFramework)' == '$(TargetFrameworkNet6Win)' Or '$(TargetFramework)' == '$(TargetFrameworkNetStandard)' Or '$(TargetFramework)' == '$(TargetFrameworkNetDesktop462)' ">
```

Comment:
I would just add it each individual ItemGroup (there is one already for each of them).